README.md

```diff
@@ -56,6 +56,11 @@ Please also check the documentation of [Language Support for Java by Red Hat](ht
 - `projectName` - The preferred project in which the debugger searches for classes. There could be duplicated class names in different projects. This setting also works when the debugger looks for the specified main class when launching a program.
 - `cwd` - The working directory of the program.
 - `env` - The extra environment variables for the program.
+- `debugFilters` - Customize the filters to debug "Just My Code".
+  - `stepFilters` - Skip these classes when stepping. Each filter could be fully qualified class name or pattern. Example: [\"java.lang.ClassLoader\", \"java.*\"].
```

Comment:
Should be more detail information about the pattern that can be applied here? Regex?